src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && (
+          <ZapMessageContainer onClick={listItems?.length > 1 ? openSearchList : undefined}>
+            ⚡ Or <HighlightText> select a token </HighlightText> in your wallet to ZAP
```

Comment:
nit: This highlighted text looks like a link to me, is that intended?

Reply:
yes, was requested by evan